src/analysis/gMCS/calculateGeneMCS.m

```diff
@@ -237,76 +259,138 @@
     lb(var.zp) = 0;
     lb(var.zw) = 0;
 
-% obj vector
+% Cplex - obj vector
     obj(var.u, 1) = 0;
     obj(var.vp) = 0;
     obj(var.w) = 0;
     obj(var.zp) = n_genes_KO;
     obj(var.zw) = 0;
 
-% ctype vector
+% Cplex - ctype vector
     ctype(var.u) = 'C';
     ctype(var.vp) = 'C';
     ctype(var.w) = 'C';
     ctype(var.zp) = 'B';
     ctype(var.zw) = 'B';
 
-% Introduce all data in a structure
-    MILPproblem.A = A;
-    MILPproblem.b = rhs;
-    MILPproblem.c = obj;
-    MILPproblem.lb = lb;
-    MILPproblem.ub = ub;
-    MILPproblem.csense = csense;
-    MILPproblem.vartype = ctype;
-    MILPproblem.osense = 1;
-    MILPproblem.x0 = [];
+% Cplex - sense of the optimization
+    sense = 'minimize';
+    
+% Cplex - Introduce all data in a Cplex structure
+    cplex = Cplex('geneMCS');
+    cplex.Model.A = A;
+    cplex.Model.rhs = rhs;
+    cplex.Model.lhs = lhs;
+    cplex.Model.ub = ub;
+    cplex.Model.lb = lb;
+    cplex.Model.obj = obj;
+    cplex.Model.ctype = ctype;
+    cplex.Model.sense = sense;
+
+% Cplex Indicators
+    % z = 1  -->  v >= alpha
+    for ivar = 1:length(var_group.z)
+        a = zeros(n_vars, 1);
+        a(var_group.v(ivar)) = 1;
+        cplex.addIndicators(var_group.z(ivar), 0, a, 'G', alpha);
+    end
+
+% Cplex Indicators
+    % z = 0  -->  v <= 0
+    for ivar = 1:length(var_group.z)
+        a = zeros(n_vars, 1);
+        a(var_group.v(ivar)) = 1;
+        cplex.addIndicators(var_group.z(ivar), 1, a, 'L', 0);
+    end
+
+% Cplex Parameters
+    cplex.Param.mip.tolerances.integrality.Cur = integrality_tolerance;
```

Comment:
Is there a way you could use the [setCplexParam](https://github.com/opencobra/cobratoolbox/blob/master/src/base/solvers/setCplexParam.m) function?